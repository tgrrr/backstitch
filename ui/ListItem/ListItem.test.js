import React from 'react'
import renderer from 'react-test-renderer'
import { ListItem } from '.'

it('ListItem: default', () => {
    const component = renderer.create(
        <ListItem>
            <>
                ListItem
            </>
        </ListItem>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
