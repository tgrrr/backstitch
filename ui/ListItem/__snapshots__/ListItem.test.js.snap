// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Listitem "
    data-testid="Listitem"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      ListItem
    </li>
  </div>
</div>
`;
